lib/teselagen/api/design_client.py

```diff
@@ -191,6 +191,29 @@ def get_design(self, design_id: Union[int, str])->dict:
         out = json.loads(response["content"])
         return out
 
+    @requires_login
+    def post_design(self, design:dict, allow_duplicates:bool=False):
+        """ Sumbits a new design into DESIGN module
+
+        Args:
+            design_dict (dict): A dictionary with the design. This dictionary
```

Comment:
Replace `design_dict` with `design` in docstring

Reply:
Done!